README.md

```diff
@@ -28,33 +31,27 @@ The Earthquake Monitor project continuously ingests real-time earthquake data fr
 1. **Docker + ECR for ETL Deployment:**
 Using Docker allows for consistent development and runtime environments. Pushing the ETL to Amazon ECR ensures seamless integration with AWS Lambda, improving deployment speed and scalability.
 
-2. **AWS Lambda for ETL Processing:**
-Lambda provides a serverless, cost-effective solution for running the ETL every minute, ensuring high availability and automatic scaling without server management overhead.
+2. **AWS Lambda:**
+Lambda provides a serverless, cost-effective solution for running the ETL, short-to-long term data storage processing and alert notification via SNS, ensuring high availability and automatic scaling without server management overhead.
 
-3. **EventBridge Scheduler:**
+1. **EventBridge Scheduler:**
```

Comment:
Are the bullet points from here onwards numbered correectly?

Reply:
it looks weird in the markdown but fine in the preview.